public/app/features/plugins/extensions/logs/log.ts

```diff
@@ -0,0 +1,94 @@
+import { isString } from 'lodash';
+import { nanoid } from 'nanoid';
+import { Observable, ReplaySubject } from 'rxjs';
+
+import { Labels, LogLevel } from '@grafana/data';
+
+export type LogItem = {
```

Comment:
Nit: maybe `ExtensionsLogItem` would be more specific.
```suggestion
export type ExtensionsLogItem = {
```